datahub-web-react/src/app/ingest/source/builder/RecipeForm/utils.tsx

```diff
@@ -202,6 +208,89 @@ export const REDSHIFT_PASSWORD: RecipeField = {
     rules: null,
 };
 
+export const TABLEAU_CONNECTION_URI: RecipeField = {
+    name: 'connection_uri',
+    label: 'Connection URI',
+    tooltip: 'Tableau host URL.',
+    type: FieldType.TEXT,
+    fieldPath: 'source.config.connection_uri',
```

Comment:
my bad I think I gave you a typo here - everything that says `connection` should be `connect`

so `connection_uri` -> `connect_uri`

Reply:
Checked & Fixed